Assets/Scripts/Statistics/Snapshots/CommunicationSnapshot.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Maes.Statistics.Snapshots
 {
-    public readonly struct CommunicationSnapshot
+    public sealed class CommunicationSnapshot
```

Comment:
Why? This allocates a lot. Which I explicitly wanted to avoid

Reply:
Yeah, i know. It is because of the data processor.
Im debugging the issue. But it seems that the reading part of the CSV helper library is not happy with readonly structs.
I'll resort to this if it is neccesary. 

But cool didnt know you could use it like that.